src/api/transactionSubmission/build.ts

```diff
@@ -31,12 +32,14 @@ export class Build {
    *
    * @returns SimpleTransaction
    */
-  async simple(args: {
-    sender: AccountAddressInput;
-    data: InputGenerateTransactionPayloadData;
-    options?: InputGenerateTransactionOptions;
-    withFeePayer?: boolean;
-  }): Promise<SimpleTransaction> {
+  async simple(
+    args: {
+      sender: AccountAddressInput;
+      data: InputGenerateTransactionPayloadData;
+      options?: InputGenerateTransactionOptions;
+      withFeePayer?: boolean;
+    } & OptionalAbi,
```

Comment:
I would just define it in the `InputGenerate....RawTransactionData` instead of a new type definition 
```
abi?: EntryFunctionABI
```